crates/wasmtime/src/func/typed.rs

```diff
@@ -207,7 +207,7 @@ unsafe impl WasmTy for Option<ExternRef> {
             unsafe {
                 store
                     .externref_activations_table()
-                    .insert_with_gc(x.inner, store.stack_map_registry());
+                    .insert_with_gc(x.inner, &*store.stack_map_lookup());
```

Comment:
The leading `&*` I think can be removed here

Reply:
Thanks, forgot to update as I changed `stack_map_lookup`'s return type.